app/lib/constituency_api.rb

```diff
@@ -6,9 +6,25 @@ class Constituency < Struct.new(:name, :mp)
 
   class Mp < Struct.new(:id, :name, :start_date)
     URL = "http://www.parliament.uk/biographies/commons"
+
+    def initialize(id, name, start_date)
+      self.id = id.to_i
+      self.name = name
+      self.start_date = parse_date(start_date)
```

Comment:
Is the start_date ever nil? Or anything else? Seems to me that an MP must have a start date otherwise they'd wouldn't be in the list. If it blows up with an ArgumentError and we get an Airbrake notification then that'd be a good indicator that's something's changed about the API - rescuing it and setting it to nil seems likely to mask possible issues.